athena-example/README.md

```diff
@@ -279,3 +279,8 @@ describe `lambda:<function_name>`.schema1.table1;
 *note that the <function_name> corresponds to the name of your Lambda function.*
 
 Don't forget to try out [Amazon Athena ML](https://github.com/awslabs/aws-athena-query-federation/wiki/AthenaML-Tutorial) to apply SageMaker model inference right from your Amazon Athena queries.
+
+## Integration Tests
+
+Although the topic of integration tests is not covered in this example, you can learn more about how to create and
+run integration tests for your connector in the [Integration-Test Module](https://github.com/awslabs/aws-athena-query-federation/blob/master/athena-federation-integ-test/README.md).
```

Comment:
Is it possible to add an example here? i think this connector reads from S3 so maybe we can write an example for it?

Reply:
its ok to do this as a separate PR but i think it is important.

Reply:
Let's target this for a future enhancement.